.github/workflows/clang-tests.yml

```diff
@@ -39,8 +42,16 @@ jobs:
       uses: hendrikmuhs/ccache-action@v1
       with:
         key: ${{ matrix.os }}
-    - name: Test clang
-      uses: llvm/actions/build-test-llvm-project@main
-      with:
-        cmake_args: -G Ninja  -DLLVM_ENABLE_PROJECTS="clang" -DCMAKE_BUILD_TYPE=Release -DCMAKE_C_COMPILER_LAUNCHER=ccache -DCMAKE_CXX_COMPILER_LAUNCHER=ccache
-        build_target: check-clang
+    - name: Test clang macOs
```

Comment:
Why isn't the name of the operating system spelled the right way, i.e. `macOS`?

Reply:
a typo. fixed.